.github/workflows/pull_requests.yml

```diff
@@ -23,9 +23,26 @@ jobs:
         - image: swiftlang/swift:nightly-main-jammy
     container:
       image: ${{ matrix.swift.image }}
+    timeout-minutes: 20
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
     - name: Run tests
       run: swift test
-      timeout-minutes: 20
\ No newline at end of file
+
+  api-breakage:
```

Comment:
Small UI thing, should we give each job a more human readable `name`? E.g. "Unit tests", "Check for API breaking changes". This would be displayed in the checks on the PR rather than "unit-tests" or "api-breakage".

Reply:
Don't think that's possible since the following restriction for job ids apply

> The <job_id> must start with a letter or _ and contain only alphanumeric characters, -, or _.

Reply:
Yup, but you can add a `name:` key as well which will be used as the "pretty" job name, e.g. https://github.com/grpc/grpc-swift/blob/43aa501c505b0447291755fd350914c1df4e94bc/.github/workflows/ci.yaml#L9

Reply:
Ah amazing. Updated!